quarkus/extensions/kogito-quarkus-serverless-workflow-extension/kogito-quarkus-serverless-workflow-deployment/src/main/java/org/kie/kogito/quarkus/serverless/workflow/deployment/ServerlessWorkflowAssetsProcessor.java

```diff
@@ -38,4 +42,9 @@ void addExpressionHandlers(BuildProducer<ServiceProviderBuildItem> serviceProvid
         serviceProvider.produce(ServiceProviderBuildItem.allProvidersFromClassPath(ExpressionHandler.class.getCanonicalName()));
     }
 
+    @BuildStep
+    void addOpenAPIWorkItemHandler(KogitoBuildContextBuildItem contextBI, CombinedIndexBuildItem indexBuildItem, BuildProducer<KogitoAddonsGeneratedSourcesBuildItem> sources) {
```

Comment:
Just to keep the naming consistency.

```suggestion
    void addOpenApiWorkItemHandler(KogitoBuildContextBuildItem contextBI, CombinedIndexBuildItem indexBuildItem, BuildProducer<KogitoAddonsGeneratedSourcesBuildItem> sources) {
```